<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #004080; /* Dark blue text */
    font-size: 16px; /* Adjust font size as needed */
}
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#004080;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #E5F3FF; /* Lighter blue background for table */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
}
QTableWidget::item {
    padding: 8px; /* Adjust padding as needed */
}
QHeaderView::section {
    background-color: #004080; /* Dark blue background for table header */
    color: #FFFFFF; /* White text color for table header */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Search Results</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #004080; /* Dark blue background for buttons */
    color: #FFFFFF; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 16px; /* Adjust padding as needed */
}

QPushButton:hover {
    background-color: #0059b3; /* Lighter blue on hover */
}

QPushButton:pressed {
    background-color: #00264d; /* Darker blue when pressed */
}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    background-color: #CCE5FF; /* Light blue background for list widget */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Adjust horizontal and vertical padding as needed */
}

QListWidget::item {
    padding: 5px 0; /* Adjust vertical padding for list items */
color: #004080; /* Dark blue text color for list items */
}</string>
   </property>
   <item>
    <property name="text">
     <string>Class</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Student</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Professor</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="search_for_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    color: #004080; /* Dark blue text */
    font-size: 16px; /* Adjust font size as needed */

</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#004080;&quot;&gt;Search for:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>133</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #004080; /* Dark blue background for buttons */
    color: #FFFFFF; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 16px; /* Adjust padding as needed */
}

QPushButton:hover {
    background-color: #0059b3; /* Lighter blue on hover */
}

QPushButton:pressed {
    background-color: #00264d; /* Darker blue when pressed */
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
